net._86box._86Box.yaml

```diff
@@ -0,0 +1,103 @@
+app-id: net._86box._86Box
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: 86Box.sh
+finish-args:
+  - --share=network
+  - --share=ipc
+  # 86Box has issues on Wayland
```

Comment:
IMO these issues are pretty minor. Are you sure it worth sacrificing wayland's security advantages and having that red "unsecure" label in GNOME Software?

Reply:
I think it is worth it. The 86Box window had some weird glitches and other issues (for example when moving the window or switching to fullscreen) when using native Wayland on my GNOME Wayland session. And if only a software renderer works on Wayland, it is quite a big issue.

Reply:
Well, the software renderer is the default anyway (I suppose it's the most stable one?). This should be fixed in 86Box in the end.

Reply:
Should be, but I think that it is a good idea to disable the Wayland socket until it actually gets fixed.

Reply:
It looks like that the Vulkan renderer is not working also on x11.

Reply:
Hmm, it works if ran with `--devel`, i.e. on Sdk instead of Platform. Looks like the app improperly tries to open (`dlopen()`?) some development library.

Reply:
86Box's OpenGL and Vulkan renderers works just fine here with LabWC and with `wayland` platform. This is strictly a runtime issue as the application dlopens neither `libGL.so` nor `libOpenGL.so` on its own and explicitly avoids dynamically linking against Vulkan.